addons/hw_posbox_homepage/views/homepage.html

```diff
@@ -110,6 +110,26 @@ <h2 class="text-center text-green">Your IoT Box is up and running</h2>
             <td class="heading">Server</td>
             <td><a href='{{ server_status }}' target=_blank>{{ server_status }}<a class="btn btn-sm float-right" href='/server'>configure</a></td>
         </tr>
+        <tr>
+            <td class="heading">HTTPS certificate
+            {% if not is_certificate_ok %}
+                <br/>⚠
+            {% endif %}
+            </td>
+            <td>
+                {% if is_certificate_ok %}
+                    OK
+                    <button class="btn btn-sm float-right" onclick="alert('{{ certificate_details }}')">details</button>
```

Comment:
@xmo-odoo 
I do agree to some extend with you on the matter
> Maybe use some sort of foldable control instead? It seems weird that the details appear in an unwiedly alert in the success case and are more accessible in the error case.

The HTTPS certificate should be pretty "transparent" for client in my opinion. Knowing the certificate details is likely too technical for IoT users, Thus the information should not be displayed directly.

Also, the certificate will automatically renew itself on expiration, if we show the date right away, I'm afraid that client will complain on the fact that it will expire in X days (because they don't know that it automatically refresh itself).

> Especially when 9 times out of 10 it'll just report the cert validity, and 1/10 it'll tell you the certificate was created correctly (but for some reason will neither provide the certificate's CN nor its validity, so I assume you have to reload the homepage and then reclick on this to get the info).

I don't really understand your point here 🤔 the button will be next to the OK and can be clicked to show the alert message in the browser. If they had an issue before, they would indeed need to reload before to update the certificate state.

> And from a UX perspective this could use some bootstrap background color thing to make the state of the certificate clearer at a glance.
There is no bootstrap on the IoT box homepage :( I indeed thought to put some yellow when it was wrong or something like this.

Reply:
> Sure I'm not saying to show it by default, but showing it in an alert hidden behind a button feels weird and off-putting. You could use a [<details>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) instead so it's folded out e.g.

Didn't know about that, I like the idea thanks :) 


> Even though the certificate is valid (again), you have to reload (?) to access the "normal" success state and get the certificate information.

Indeed, I did change the code on the matter using a recursive call, what do you think ? I'm not a big fan of using recursion in this case, but defining a new function or copy-pasting the code was looking less ideal for this case. Let me know if you would see a better way to do so :)